frontend/src/modules/application/components/ApplicationForm.jsx

```diff
@@ -0,0 +1,114 @@
+import React from "react"
+import { personalInformationFields, currentProgramInformationFields, currentStatusFields, customQuestions } from "./FormFields"
+
+const DefaultInput = ({ label, curValue, onChange, required }) => <div>
+    <label style={{ width: "20vw" }}>{required ? label + ':' : label}</label>
+    <input style={{ width: "30vw" }} value={curValue} onChange={onChange} required={required} />
+</div>
+
+const TextboxInput = ({ label, curValue, onChange, required }) => <div>
+    <label style={{ width: "20vh" }}>{required ? label + ':' : label}</label>
+    <textarea style={{ width: "30vw" }} 
+              value={curValue} 
+              onChange={onChange} 
+              required={required}/>
+</div>
+
+const BinaryRadioInput = ({ label, curValue,required }) => <div>
+    <label style={{ width: "20vw" }}>{required ? label + ':' : label}</label>
+    <label>
+        <input type="radio" checked={curValue === "true"}/>
+        Yes
+    </label>
+    <label>
+        <input type="radio" checked={curValue === "false"}/>
+        No
+    </label>
+</div>
+
+
+
+const allFields = [].concat.apply([], 
+    [personalInformationFields, 
+    currentProgramInformationFields,
+    currentStatusFields, 
+    customQuestions])
```

Comment:
If I understand your intentions correctly, you're trying to concatenate these various arrays into one ? You can simplify this code; concat allows you to achieve this by passing several parameters, i.e. `[].concat([1], [2],[3])`